client/src/components/BoardObject.tsx

```diff
@@ -91,10 +92,10 @@ export const BoardObject = (props: BoardObjectProps) => {
 			>
 				{type === 'gif' && data && <Gif gif={data} width={180} noLink={true} />}
 				{type === 'image' && imgSrc && (
-					<img alt="user-selected-img" src={imgSrc} style={{ width: 180 }} />
+					<img alt="user-selected-img" src={imgSrc} style={{ width: 180 }}/>
```

Comment:
notice how these are whitespace changes, not actual code changes. You should run the command in client `npm run prettier` to format the code to keep consistent styles, or else it's hard to tell what you changed